browser/src/control/Control.NotebookbarCalc.js

```diff
@@ -1027,10 +1027,32 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 				'command': _('Toggle UI Mode')
 			},
 			{
-				'id': 'showstatusbar',
-				'type': 'menubartoolitem',
-				'text': _('Status Bar'),
-				'command': _('Show Status Bar')
+				'type': 'container',
+				'children': [
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'showstatusbar',
+								'type': 'menubartoolitem',
+								'text': _('Status Bar'),
+								'command': _('Show Status Bar')
+							}
+						]
+					},
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'collapsenotebookbar',
+								'type': 'menubartoolitem',
+								'text': _('Tabbed Bar'),
```

Comment:
I'm not sure for the label : `Tabbed Bar` vs `Tab Bar` vs `Menu Bar` vs `Main Bar` ?

Reply:
hm.. right. At first sight I would say Tab Bar but we do understand why you have used the wording `Tabbed` (we use `Tabbed view` for that menu entry when using compact view). Still I think it can be misleading to name that button with any of those options since it looks like it's a toggle but in fact for the user is more of a single action: Collapse Tabs